spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationPropertiesProvider.scala

```diff
@@ -130,12 +130,21 @@ class StandardizationPropertiesProvider {
       val isXcomOpt = if (cobolOptions.isXcom) Some(true) else None
       val isTextOpt = if (cobolOptions.isText) Some(true) else None
       val isAscii = cobolOptions.encoding.exists(_.equalsIgnoreCase("ascii"))
+
```

Comment:
With these changes and removal of `"is_record_sequence"` and `"is_text"` from the `HashMap` I believe `isXcomOpt` and `isTextOpt` doesn't make much sense anymore.

Reply:
Took me some time to understand what you mean (initially I thought you were talking about `isXcom` and `isText` from `CobolOptions`) :D I agree, removed them.

Reply:
Sorry, if it was not clear. Next time don't hesitate to ask. 😉